### DISCUSSION TITLE
we need wechat support

### DISCUSSION THREAD

**Discussion by kulame** [nextauthjs/next-auth#6236] [Ideas]
### Description 📓

WeChat is the largest social network in China. Many applications in China rely on WeChat login. We strongly hope that nextauth will support WeChat auth

### How to reproduce ☕️

11

### Contributing 🙌🏽

Yes, I am willing to help implement this feature in a PR

**Comment by jonidelv** [comment]
Looking forward to WeChat support. Is this PR too incomplete for integrating WeChat https://github.com/nextauthjs/next-auth/pull/2519 ?
